internal/cli/atlas/integrations/create/webhook.go

```diff
@@ -58,11 +58,9 @@ func (opts *WebhookOpts) Run() error {
 
 func (opts *WebhookOpts) newWebhookIntegration() *atlasv2.ThridPartyIntegration {
 	return &atlasv2.ThridPartyIntegration{
-		Webhook: &atlasv2.Webhook{
-			Type:   &webhookIntegrationType,
-			Url:    opts.url,
-			Secret: &opts.secret,
-		},
+		Type:   &webhookIntegrationType,
```

Comment:
With merge of the fields we lost information about what fields are part of the object.
I have POC that will help us to flag objects in the SDK: https://github.com/mongodb/atlas-sdk-go/pull/117